code/modules/mob/living/carbon/resist.dm

```diff
@@ -69,12 +69,12 @@
 			if(GRAB_PASSIVE)
 				qdel(G)
 			if(GRAB_AGGRESSIVE)
-				if(prob(60)) //same chance of breaking the grab as disarm
+				if(prob(max(60 + ((stats?.getStat(STAT_ROB)) - G.assailant?.stats.getStat(STAT_ROB) ** 0.8), 1))) // same scaling as cooldown increase and if you manage to be THAT BAD, 1% for luck
```

Comment:
bad change since it makes grabs even more stronger than they are currently- i can't see any way of this being good unless it's basically a nonfactor in resisting out of grabs

Reply:
Currently, when in a neck grab your chance to escape is completely random as i understand. This change means that bigger, stronger individuals have more pull others around if they're particularly wimpy and vice versa. The increased time to upgrade grabs is also a good move away from grappling being 2good

Reply:
yeah the grabs requiring time to upgrade is a good change but the original numbers on the eris pr were really harsh for the resist changes